addons/html_editor/static/src/main/font/font_plugin.js

```diff
@@ -244,6 +244,10 @@ export class FontPlugin extends Plugin {
             !nodesAfterTarget.length ||
             (nodesAfterTarget.length === 1 && nodesAfterTarget[0].nodeName === "BR")
         ) {
+            const closestLi = closestElement(targetNode, 'li');
+            if (closestLi) {
+                return;
+            }
```

Comment:
Why choose `handleSplitBlockPRE` for the fix. I think it's just about returning from `handleSplitBlock` in `list_plugin` if `targetNode` is `PRE`.